flow-typed/Claim.js

```diff
@@ -66,6 +66,7 @@ declare type CollectionId = ClaimId | string; // uuid
 
 declare type ChannelId = ClaimId;
 
```

Comment:
:small_red_triangle: Can't add tag to new playlist

Create a new private playlist, the add tags when trying to publish.  Clicking on the tag suggestion cloud does nothing.